edge_orchestrator/Dockerfile

```diff
@@ -5,8 +5,9 @@ WORKDIR /edge_orchestrator
 COPY pyproject.toml ./pyproject.toml
 COPY edge_orchestrator ./edge_orchestrator
 
-RUN apt-get update
+RUN apt-get update -y
 RUN apt-get install fswebcam -y
+RUN apt-get install libpq-dev zlib1g-dev jq gcc libnuma-dev -y
```

Comment:
ajouter un commentaire pour dire que libpq c'est pour éviter l'erreur "psycopg2.OperationalError: SCRAM authentication requires libpq version 10 or above"

Reply:
SCRAM is important ^^